docs/data-sources/registered_model_versions.md

```diff
@@ -0,0 +1,64 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_registered_model_versions Data Source
+
+-> This resource can only be used with a workspace-level provider!
+
+This resource allows you to get information about versions of [Model in Unity Catalog](https://docs.databricks.com/en/mlflow/models-in-uc.html).
+
+## Example Usage
+
+```hcl
+data "databricks_registered_model_versions" "this" {
+  full_name = "main.default.my_model"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `full_name` - (Required) The fully-qualified name of the registered model (`catalog_name.schema_name.name`).
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `model_versions` - list of objects describing the model versions (only filled if `include_model_versions` is set to `true`). Each object consists of following attributes:
```

Comment:
what is `include_model_version` argument here?

Reply:
ouch, copy paste from the previous implementation